packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -339,13 +328,15 @@ export class SDKClient {
       return transactionResponse;
     } finally {
       if (transactionId?.length) {
-        this.eventEmitter.emit(constants.EVENTS.EXECUTE_TRANSACTION, {
+        this.eventEmitter.emit('execute_transaction', {
           transactionId,
-          requestDetails,
+          callerName,
```

Comment:
We removed `callerName` and `interactingEntity` from this metric, we should remove them in `IExecuteTransactionEventPayload` interface instead.